src/components/table/table-cell/table-cell.component.js

```diff
@@ -19,14 +20,41 @@ import OptionsHelper from '../../../utils/helpers/options-helper';
  * set styling options for the cell used for action such as delete.
  */
 class TableCell extends React.Component {
+  /**
+   * Returns if child element has type property.
+   */
+  childHasType(child) {
+    return child && child.type;
+  }
+
+  /**
+   * Returns if child element is Textarea.
+   */
+  childrenHasTextArea() {
+    const { children } = this.props;
+
+    if (!Array.isArray(children)) {
+      return this.childHasType(children) && children.type.name === TextArea.name;
```

Comment:
With the suggestion above this would be simplified to:
`return this.childType(children) === TextArea.name`

Reply:
Yeah good point I'll make the change